app/views/avo/actions/show.html.erb

```diff
@@ -51,7 +51,8 @@
             size: :sm,
             data: {
               target: :submit_action,
-            } do %>
+              action_target: "submit"
+            }, autofocus: (@action.get_fields.count == 2 ? true : false)  do %>
```

Comment:
`get_fields` will fetch the fields again, that's not necessary, actually we are already fetching them twice. Lets fetch them on `Avo::ActionsController` like: `@fields = @action.get_fields` and replace all `@action.get_fields` with `@fields` on the view.

Reply:
Good idea. I will make the changes.